packages/component-library/draft/Kaizen/Select/Select.tsx

```diff
@@ -64,7 +64,7 @@ const Menu: typeof components.Menu = props => (
 const Option: typeof components.Option = props => (
   <components.Option
     {...props}
-    className={classNames(styles.option, {
+    className={classNames("needsclick", styles.option, {
```

Comment:
Can you drop a TODO in here? Something like:

`TODO - needsclick class disables fastclick on this element. Remove when fastclick is removed from consuming repos`